app/screens/ChatbotScreen.tsx

```diff
@@ -266,19 +275,25 @@ const ChatbotScreen = ({
       {/* Chatbot pages screen */}
       {!isLoading && pages && pageIndex >= 0 && pageIndex < pages.length && (
         <View style={styles.container}>
-          <View style={{flex: 1}}>
-            {/* BACK & PROGRESS */}
-            <View style={styles.progressHeader}>
-              <IconButton color={Colors.Primary} icon="arrow-left" onPress={() => { onBack() }} />
-              {/* <SpacerH />
-              <View style={styles.progressBar}>
-                <ProgressBar color={Colors.Primary} progress={progress} />
-              </View> */}
-            </View>
-            <SpacerV />
+          {/* BACK & PROGRESS */}
+          <View style={styles.progressHeader}>
+            <IconButton color={Colors.Primary} icon="arrow-left" onPress={() => { onBack() }} />
+            {/* <SpacerH />
+            <View style={styles.progressBar}>
+              <ProgressBar color={Colors.Primary} progress={progress} />
+            </View> */}
+          </View>
+          <SpacerV />
+          <ScrollView ref={scrollViewRef} contentContainerStyle={{flex: 1}}>
             {/* HEADER */}
             {pages[pageIndex].header !== undefined && (
-              <H2 text={pages[pageIndex].header ?? ""} />
+              <>
+                {height > 667 ? (
```

Comment:
I was about to say _"…this is normally done via "CSS" or in this case styling"_, but NOPE, media queries and sorts are not supported on react native 😮. So basically having to use javascript for a CSS native feature (which browsers can't cache or performance improve on, also no ServerSideRendering, which we don't use.)… interesting. One caveat of react-native:web over standard react 🤔.

Consider this a non issue 😆